integrations/airflow/marquez_airflow/extractors/great_expectations_extractor.py

```diff
@@ -79,6 +79,11 @@ def _get_schema() -> str:
         return "https://github.com/MarquezProject/marquez/tree/main/integrations/airflow/marquez_airflow/extractors/ge-assertions-dataset-facet.json"  # noqa
 
 
+def provide_dataset_info(operator, namespace: str, name: str):
```

Comment:
Mind if we add docs on the importance of setting the `_expectations*` mappings? And I'd call the method `set_dataset_info()` given that it will be called to mutate great expectations values for the extractor.

Reply:
I've provided info in integration's readme, and now added docstring with example to the function.